src/Symfony/Component/Serializer/Attribute/SerializeFormatter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Attribute;
+
+/**
+ * Defines a callable that will be used to format the property data during serialization.
+ *
+ * The first argument of that callable must be the input data.
+ * Then, it is possible to inject the {@see Symfony\Component\Serializer\Serialize\Config\SerializeConfig}
+ * and services thanks to their FQCN.
```

Comment:
this is a very nice feature!
do you you provide a way to specify a service name, instead of being based on the FQCN?

Reply:
Actually, you can use `Autowire` attributes here, so you can indeed specify the service name explicitly :slightly_smiling_face: